carnival.cabal

```diff
@@ -146,6 +146,64 @@ executable         carnival
 
     ghc-options:       -threaded -O2 -rtsopts -with-rtsopts=-N
 
+executable carnival-integration
```

Comment:
I don't know of a way to centralize extensions, etc.

We can audit the build depends, but I suspect we need them all b/c we
import TestImport which references a lot. Might we worth it to duplicate
what we need in IntegrationImport or break up the modules differently. Then
again, the depends list isn't a huge deal...
On May 1, 2015 11:28 AM, "Joe Ferris" <notifications@github.com> wrote:

> In carnival.cabal
> <https://github.com/thoughtbot/carnival/pull/250#discussion_r29508231>:
>
> > @@ -146,6 +146,64 @@ executable         carnival
> >
> >      ghc-options:       -threaded -O2 -rtsopts -with-rtsopts=-N
> >
> > +executable carnival-integration
>
> Bummer that we need to repeat a lot of these options. Is there a way to
> specify extension or ghc-options for every target? Do we need all of
> these build-depends or are some of them only used from the library?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/carnival/pull/250/files#r29508231>.
>